test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
So I found a different workaround using `tmp` which is pretty much the following.

```js
var tmpFile = tmp.fileSync();
var name = process.platform === 'win32' ? tmpFile.name.split('\\').join('\\\\') : tmpFile.name;
```

On a scale of 1-10 how much do you hate it? lul